docs/readmes/nms/federation.md

```diff
@@ -0,0 +1,55 @@
+---
+id: federation
+title: Federation
+hide_title: true
+---
+# Federated Networks
+
+Magma implements a low-friction, single point of integration between an MNO's mobile core and access gateways through federation networks. NMS enables management of the federated networks and their gateways.
+
+Magma has two important concepts on federation:
+
+### **Federation Network/** Federation Gateway (FGW)
+
+Federation network comprises of the federation gateways. The Federation Gateway integrates the Operators core network with using standard 3GPP interfaces to existing MNO components. It acts as a proxy between the Magma AGW and the operator’s network and facilitates core functions, such as authentication, data plans, policy enforcement, and charging.
+
```

Comment:
what about adding 3gpp interfaces names so the reader that are used to 3gpp terminology can have an anchor point?

```
S6a, gx, gy, S8
```